Lib/glyphsLib/builder/features.py

```diff
@@ -96,9 +96,7 @@ def _to_ufo_features(font, ufo=None, generate_GDEF=False, skip_export_glyphs=Non
                 name = feature_name.groups()[-1]
                 lines.extend(name.splitlines())
             else:
-                feature_name = re.search(
-                    "^Name: (.+)", feature.notes, flags=re.MULTILINE
-                )
+                feature_name = re.search("^Name: (.+)", feature.notes)
```

Comment:
Do you think the name should be removed from the notes comment after it has been written to the UFO feature code?

If it is left in the notes comment, we get code like this in the UFO:

```afdko
feature ss01 {
# notes:
# Name: Alternate g
featureNames {
  name "Alternate g";
};
# automatic
sub g by g.ss01;
} ss01;
```

Reply:
Yes I think we should avoid the duplication

Reply:
Ok